The build was failing due to warnings on these lines.
---

diff --git a/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/lib.rs b/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/lib.rs
index 1b72962..3dbdb49 100644
--- a/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/lib.rs
+++ b/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/lib.rs
@@ -216,7 +216,7 @@ impl QrCode {
     }
 
     /// Returns an iterator over QR code vector of colors.
-    pub fn iter(&self) -> QrCodeIterator {
+    pub fn iter(&self) -> QrCodeIterator<'_> {
         QrCodeIterator::new(self)
     }
 
diff --git a/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/optimize.rs b/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/optimize.rs
index 26e9940..8a8bde9 100644
--- a/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/optimize.rs
+++ b/third_party/rust/chromium_crates_io/vendor/qr_code-v2/src/optimize.rs
@@ -109,7 +109,7 @@ impl<'a> Parser<'a> {
     ///                                Segment { mode: Numeric, begin: 3, end: 6 },
     ///                                Segment { mode: Byte, begin: 6, end: 10 }]);
     ///
-    pub fn new(data: &[u8]) -> Parser {
+    pub fn new(data: &[u8]) -> Parser<'_> {
         Parser {
             ecs_iter: EcsIter {
                 base: data.iter(),
